src/common/MessageDispatcher.sol

```diff
@@ -396,7 +396,7 @@ contract MessageDispatcher is Auth, IMessageDispatcher {
         uint128 remoteExtraGasLimit
     ) external auth returns (uint256 cost) {
         if (poolId.centrifugeId() == localCentrifugeId) {
-            return hub.initiateTransferShares(
+            return spokeHandler.initiateTransferShares(
```

Comment:
Seeing this, not sure if it should be called hubHandler 🤔, maybe people think it belongs to `spoke` by the name?